lib/edit-site-page.php

```diff
@@ -143,11 +152,31 @@ function gutenberg_edit_site_init( $hook ) {
 		)
 	);
 
-	// Preload server-registered block schemas.
 	wp_add_inline_script(
 		'wp-blocks',
-		'wp.blocks.unstable__bootstrapServerSideBlockDefinitions(' . wp_json_encode( get_block_editor_server_block_settings() ) . ');'
+		sprintf( 'wp.blocks.setCategories( %s );', wp_json_encode( get_block_categories( $post ) ) ),
+		'after'
 	);
+	
+	/**
+	 * Scripts
+	 */
+	wp_enqueue_media();
+	wp_tinymce_inline_scripts();
+	wp_enqueue_editor();
```

Comment:
Why do we need these?

Reply:
@youknowriad Good catch. Those were leftovers from my testing. They are not needed. I will remove those.